### PR TITLE
Added html-proofer gem.

### PR DESCRIPTION
**Pull request by iBobik**
Added html-proofer to all tags.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ do file a ticket and they will be corrected if possible.
 * RedCloth
 * maruku
 * pry
```
Comment (iBobik) [inline README.md]: @envygeeks Btw, there is "pry" mentioned, but I did not found it anywhere else.

**Comment by envygeeks** [comment]
You'll need to rebase this.

**Comment by iBobik** [comment]
Seems ok.

**Comment by envygeeks** [comment]
There are conflicts still and it won't let me merge from the GUI.

**Comment by iBobik** [comment]
Sorry, I propably need to go to sleep for today. Fixed.

**Pull request merged**